test/cmocka/src/math/arithmetic/base_e_logarithm.c

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <math.h>
+#include <cmocka.h>
+#include <string.h>
+
+#include <sof/math/log.h>
+#include <sof/audio/format.h>
+#include <sof/string.h>
+#include <sof/common.h>
+#include "log2_tables.h"
+/* 'Error[max] = 0.0000101176000000,THD(-dBc) = -99.8984609736932327' */
+#define CMP_TOLERANCE 0.0000101175870882
```

Comment:
Here also magic number, add comment why use this tolerance.

Reply:
Done